concepts/observability-strategy/configurable-logging/assets/logging-future.drawio.svg

```diff
@@ -0,0 +1,831 @@
rg+YYCWw8qC6YlxPhlmaXO5T5KPwSbs8WblMKOXddKtQ176GWxY29RixB62M/mo/A47UxiID+a5BjdQ9crCQEeb1c7eKMDB+31fbdQAiBHDzyh7DrIQ9/zj+/cHqfxoDgLswJ1dXnoPzx/sQP7GnXeAy7nz9U8OdecNblRuBH3eu2RmP73gqcFAhls/pb/iDKWrqGbnd2W0eZxp7duSyD04p21ti/7qv6bYuP4VTOMkLsiz7wrh/5bET9jwNu6DIgjRAv/84zrZpE/4YDKZ/D7QvLikiL4wba5k2mSWtjbzwrZfw7wYZrQfF9HPsd4rbkjJTyvlZdvnmJCqvGdv6vOw7UO6PwQFgaZyDMswBxkprtPTSLmQxMCYs2FMoeaI7yD4nEf36BnNa+z4U80q7mf9nlaVKh1oiwnoL1kwD9LgDcGiDTmvy3JFr8uVwCLUBYuuJOsUJRF97m8o3CTvMFYE+FojSUXXWeOvrhh/vVmvE0y5IkbpQBHpW2h0jsCEEEw1TWBPJNaDcVYxucL82nHWg8Jqs8NqX0/eTFRpvPI1oT/hslua1Rp1VTty+4MMZRQZFkfsJJdQ+gqOOxHhmvG8keZkPpWe4cn8sTGxLVqiOJQ7nAm2rTp/FsMsHJ/o8/5dMdj8jxyjetd/7DCH878NqZgv52SclzN4g2/w1i/lhNDL1Od8yNDLdrgJ1WGVy7Co9msHWURAn3IQC1TzIkQbUa6vn+y7tbQn1dKyiTC1TYToQ7xNI6lx8i7KRhKjmB841ntGBVPl9XGN5lneEa6J03ABsOZZUgPxI3T5mqFLKhzXErpslpNcXuhyV3C1eeSBoUyXl0zNPp8vxtQ+46O0uI2HQucllZAIVfKyaiqZBaIixyO61dTMyNdBKjU0ZhUZiJGRLabBb/gp8P/EF5ce/U4OCaGM0kKZB6s42VbdVyhFdRFd95bWiDGwESNYMCML8gvFIX450lougm7O6MvC8nVxyz05Jg8GyWtCTKND95rNvXSOTxrGaoep6NlcaRUQaZKpINzeKCFyj1G2cIqIXDDLC60yIm2lOiLNJfO3d7UCQJ8EzOdzazYrr3bFoLkeOlOHhN3Jb2xW6/ahF0H5+7Wia3+jUnXlI5MCHdqEVVf7GrXW6zRUmq/qBdumn92Baw3I9WLGaQle6os+E9pcauaylVXYUSuysbAINs2LTq/OcLWi4W6m6kbGSjWwQoIUTQcAW8Yztp2HIcUPssdhwL73z5S18LLhGMwXhytPGcqxF2rx6DayQksH4KW7AiUMZxUuUShQqQX6LlmVhN8EBbA7dcW5nGNTUuRsSRe5q0iKyFZaf+D9B95/4P0H3n/gvQa8F5woCdrLdo6xVOxQIAbUURElY4zoSTQriLQrpGcNKBJ61lByyIsaRmeumtmkL3+Ote+uQOe3tKhr0LxALl4yNmV7J2kqARHzGHebvEDvfEocLucEbHFGdBXliLkNNdVpPBRFzg4ocv2pMtY2eNY+534b9Lc6pHycYcIQ4/obHiye5QJNL30hwwGIB/4EOhAjCoSO79sOF+cExsQ2bNd2PYjR3zQdBUSWFKl//yquEBmUuDlajWaoqDKx4H7sq9KfLqc/nX5ZMxXZGtOScurAJZ1NVuWoJZ3N9i8al3TSbTQO5jx8PfVclu1P/M4fZAXJNysHrjuoulC/Ka6pvkPpPF5sMLhcH2JZe6XKmAAI2FVWNa8PnEK6jwCtq2X7KymyMsXcQNGWNPNrctA6ygLVLkXv3UmUug6eJCKnT407OrBvf3qZnZJTF7erw75q3xX1MMcl1nxoC7jWNxUqLOswvb4YeWyGGfBFSYZj7322ZjPDHR0Gr5g1Rf/2R1DMlteI187r4LXPjNms0lEL2Je+9+QglHYFsbBElHakFqqtAKb3Li0mBd9RGu5eqRrSlaokRRJu06AsWx+4UlUFUbm19ibdRepQkZaKYrnX2RzyeHerTX9UyY/zbth2mpqVLAi1dpiqeuulHJ7DHHOvMhPuhybHU/tqnwBfTg042e+r7/mB+A3G1flDkv0qHzb58oqjOIfXnlgGcFmFqGbriTHfRYlepWpUZxBIR0jHhZww2GJKRFdIxxK9fI2pp96r7IbFe7nCSiDJZGjzE2ml3ltRlm5vbWm9kroEetXlbtUjaMwTdZjr8/KvT4mJdUU006YHJFVBJE9rV7R/tUHkKZGfC5bo3gLtHhVwP02gqf5hJFpJmEmFRNMlXsdKNJ9MB74+iRZjPjqNniFL54cpdc5Ikil1bQggBmweMjLMMtqIZv8VJMS5QORYkg+X7cqixD7y3xaaWqdFE/SgqcQ8si8FS0+1jhwunsh/dkkdlFKpuF7WPEdinVm7OEuCPI9nI3b5YjVEyC1wxC2d5Y0nmgueyOH+pXC4faq1YHEFMvo4fP/uV3dotcZzdbH7bipQfXwOme4z1dF8styEiiJfyScFvqE0LlAWp4v3QX0o7DkrSeHror8pXVNTfVaoLmLvnn1Fi4d4HSUxJiS9sGBuo2dcLmm1bfudN3a2qxaVWUquQoj4GjRDvmuArhibZLt6paR+5QQqD1GSpQg7Pt6ngrbiWoQHVOBHj8sM6uMNdi0NYQNgfBZLSrTPnSb1TGvC6tJmY94u8SgSKOdL0Rev6HXtW1oPmxSu/NqV7Y2sa0tr+xJs+jN/ZaHfRxaUfN9E9tUFoKdKDPh7i2FdvtKhr/XtmQbzWTveFtf1tQUe5Q99bYEvfgTM/YNrx2zROq028bh/q1YMT1HT8M9qxdivssfqqZW7feCCiTtZmlBALjZXI/Z8NOuQ2AvVdGB/Bz5cxncYDhRiuXnzWRYJO19yAYy9I3XcFJle9adX6JdDLh1fVBbp6QmrU914cB3Q0I+4ndfycf19EOj1rEI4FgE9PnhAvzmvDKBED3ZHjcPF49MOS7pToAfpZolDeYwDJG7uTwGkUb2FROf2dvMI8On/&lt;/diagram&gt;&lt;/mxfile&gt;">
```

Comment:
Pls fix the spelling

- `otel-collector` => `Otel Collector`
- `datadog` => `Datadog`
- `dynatrace` => `Dynatrace`
- `k8s` => `Kubernetes`
- `telemetry operator` => I'd use title case in diagrams, `Telemetry Operator`

Also, I think you need more boxes for the background:
- The first Kyma-System Namespace is floating on a transparent layer. Add a box like for the second Kyma-System Namespace.
- There's a `Kyma Runtime2`, but where is `Kyma Runtime1`, and its background box??

<img width="772" alt="image" src="https://user-images.githubusercontent.com/76950046/181472350-cc4fff67-dad6-4eea-86de-09e2f9ff3cc1.png">